src/components/ContextMenu/ContextMenu.js

```diff
@@ -48,17 +48,21 @@ function ContextMenu({ children, zIndex }) {
             <Button
               onClick={handleBaseButtonClick}
               opened={opened}
+              disabled={disabled}
               focusRingRadius={RADIUS}
               css={`
-                color: ${opened ? theme.accent : theme.surfaceContent};
-                background: ${theme.surface};
-                border: 1px solid ${theme.border};
+                color: ${disabled
+                  ? theme.disabledContent
+                  : opened
+                  ? theme.accent
+                  : theme.surfaceContent};
+                background: ${disabled ? theme.disabled : theme.surface};
+                border: ${!disabled && `1px solid ${theme.border}`};
```

Comment:
We should do this so that it doesn’t expand as `border: false;` (which can display a warning):

```suggestion
                border: ${disabled ? '0' : `1px solid ${theme.border}`};
```